core/src/core/pmmr.rs

```diff
@@ -269,7 +269,7 @@ where
 	/// The last position in the PMMR
 	pub last_pos: u64,
 	backend: &'a mut B,
-	// only needed for parameterizing Backend
+	// only needed to parametrise Backend
```

Comment:
I'm pretty sure "parametrise" is not right?
Old and not in common use any more?

Reply:
Not right, just a missing 'e' typo

Reply:
Both are correct but I will update for ```parameterize```.